torchao/experimental/kernels/cpu/aarch64/tests/build_and_run_tests.sh

```diff
@@ -10,20 +10,55 @@ SCRIPT_DIR=$(cd -- "$(dirname -- "${BASH_SOURCE[0]}")" &> /dev/null && pwd)
 export TORCHAO_LIBRARIES=${SCRIPT_DIR}/../../../../../..
 export CMAKE_OUT=/tmp/cmake-out/torch_ao/tests
 
+target=${1:-"native"}
+
 IS_ARM64=0
+BUILD_ARM_I8MM=0
```

Comment:
Does this mean that you have not enabled the tests for kernels added in this PR?

Reply:
that's right. We need a way to specify newer macs in the CI to test this.